.ci.yaml

```diff
@@ -1,17 +1,40 @@
+# Describes the targets run in continuous integration environment.
+#
+# Flutter infra uses this file to generate a checklist of tasks to be performed
+# for every commit.
+#
+# More information at:
+#  * https://github.com/flutter/cocoon/blob/master/CI_YAML.md
 enabled_branches:
   - master
 
+platform_properties:
+  linux:
+    properties:
```

Comment:
do we need a `properties` key here? are there gonna be other keys at this same level?

Reply:
nevermind, I see this is already defined in the proto

Reply:
Unfortunately, yes. I can't find a way around it as protos do not make it possible to nest values (in this case we map->properties->map)

We could hardcode the platforms in the proto (linux_properties->map), but i'd prefer to make it easy to add new platforms (like web, linux_android)

Reply:
agreed. this is annoying, but fine.